import_export_extensions/widgets.py

```diff
@@ -292,9 +294,15 @@ def render(self, value: Model, *args, **kwargs) -> typing.Optional[str]:
         if not value:
             return None
 
+        is_storages_variable_available = hasattr(settings, "STORAGES")
+
         if (
+            not is_storages_variable_available and
             settings.DEFAULT_FILE_STORAGE
-            == "django.core.files.storage.FileSystemStorage"
+            == DEFAULT_SYSTEM_STORAGE
+            or is_storages_variable_available
+            and settings.STORAGES["default"]["BACKEND"]
+            == DEFAULT_SYSTEM_STORAGE
```

Comment:
This condition looks too complex, perhaps you can simplify it by using some helper function that returns the used file storage from the settings